### ISSUE TITLE
[prometheus-snmp-exporter] Multiple snmp targets could not be specified using ServiceMonitor

### ISSUE THREAD

**Issue by IlliaZhaglov** [prometheus-community/helm-charts#1734]
### Is your feature request related to a problem ?

There is no mechanism to specify several snmp targets using ServiceMonitor. I see two options how to overcome this issue right now but both these solutions are not ideal for me:
1) Create prometheus scrape_configs with several targets;
2) Deploy multiple releases of prometheus-snmp-exporter in order to have ServiceMonitor object for each target.

### Describe the solution you'd like.

I would like to have a possibility to specify multiple snmp targets using values.yml.

### Describe alternatives you've considered.

1) Define ServiceMonitor targets variable as a list of maps;
2) Add "range" flow control structure into servicemonitor.yaml template:
{{- range .Values.serviceMonitor.targets }}
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
...
{{- end }}
Basically, the solution is the same as it was implemented in prometheus-blackbox-exporter chart.
